examples/js/effects/VREffect.js

```diff
@@ -233,6 +233,20 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	};
 
+	this.requestAnimationFrame = function ( f ) {
+
+		if ( ! isDeprecatedAPI && vrHMD !== undefined ) {
```

Comment:
hmm you're right I saw that also. But how does that work with cancelAnimationFrame?